docs/examples/plot_KimCNN_tutorial.py

```diff
@@ -0,0 +1,131 @@
+"""
+KimCNN Model for Multi-label Classification
+===========================================
+
+This step-by-step example shows how to train and test a KimCNN model via LibMultiLabel.
+
+
+Step 1. Import the libraries
```

Comment:
We can remove "Step ??" in all sections. It's difficult to maintain the step num